app/views/plates/show.html.erb

```diff
@@ -1,18 +1,33 @@
+<a href="<%= labware_path(@plate) %>">&lt;&lt; Back to Labware <%= @plate.human_barcode %></a>
 
 <h2>QC Results</h2>
 
-<h3>Concentration</h3>
-<%= render partial: "qc_table", locals: { plate: @plate, qc_type: "concentration" } %>
+<h3>
+Concentration (ng/µl)
+<small class="text-muted">Measured from QC File upload</small>
+</h3>
+<%= render partial: "plate_schematic", locals: { plate: @plate, well_values: ['qc_result_for', "concentration"] } %>
```

Comment:
Probably not a big deal... but is having the method name as a string like this bad security practice? Could allow you to call any method on Well? 🤔 

I get what you're trying to do - it is neater like this, especially as you've added more types of data that come from 'well attribute'.